packages/ia-components/index.js

```diff
@@ -9,8 +9,8 @@ export { default as IAUXExampleComponent } from './live/example-react-component/
 export { default as HorizontalRadioGroup } from './sandbox/selectors/horizontal-radio-group/horizontal-radio-group';
 export { default as FlexboxPagination } from './sandbox/pagination/pagination-with-flexbox';
 
-export { default as TheatreTrackList } from './sandbox/theatres/components/track-list/track-list';
-export { default as TheatreAudioPlayer } from './sandbox/theatres/components/audio-player/audio-player-main';
+export { default as TheatreTrackList } from './sandbox/theatres/track-list/track-list';
+export { default as TheatreAudioPlayer } from './sandbox/theatres/audio-player/audio-player-main';
```

Comment:
let's do:
`/sandbox/track-lists/track-list`
`/sandbox/audio-players/audio-player-main`

really flatten it out